Makefile

```diff
@@ -15,15 +15,12 @@ tox:
 
 .PHONY: reformat
 reformat:
-	autoflake -ir --remove-all-unused-imports ${SOURCE_FILES}
-	isort ${SOURCE_FILES}
-	-docformatter -ir --pre-summary-newline --wrap-summaries=0 --wrap-descriptions=0 ${SOURCE_FILES}
+	ruff check . --fix
 	black .
 
 .PHONY: lint
 lint:
-	flake8 ${SOURCE_FILES}
-	pydocstyle ${SOURCE_FILES}
+	ruff . --no-fix
```

Comment:
Agreed. And we can now lose the specific source directories since both black and ruff just use `.`. Also, the shell sniffing for version can be restricted to just `make publish`.